config/application.rb

```diff
@@ -40,6 +40,9 @@ class Application < Rails::Application
     # define cache store
     config.cache_store = :file_store, Rails.root.join('tmp', "cache_file_store_#{Rails.env}")
```

Comment:
In container context i've used memcached for cache_store.
See: https://github.com/zammad/zammad-docker-compose/blob/master/containers/zammad/docker-entrypoint.sh#L58

Could we use redis here too to have a single solution for all file based caches and sessions stores?